CRM/Member/Import/Form/MapField.php

```diff
@@ -45,8 +45,6 @@ public function buildQuickForm() {
       unset($sel1['membership_id']);
     }
 
-    $sel2[''] = NULL;
```

Comment:
this is copy & paste - it never did anything but does cause some weirdness if you go forwards from the Preview screen & then back again